apps/prairielearn/src/sync/fromDisk/assessments.js

```diff
@@ -315,6 +326,76 @@ module.exports.sync = async function (courseId, courseInstanceId, assessments, q
       }
     });
   }
+
+  /** @type {Set<string>} */
+  const importedQids = new Set();
+  /** @type {Map<string, string[]>} */
+  const importedQidAssessmentMap = new Map();
+  Object.entries(assessments).forEach(([tid, assessment]) => {
+    if (!assessment.data) return;
+    (assessment.data.zones || []).forEach((zone) => {
+      (zone.questions || []).forEach((question) => {
+        let qids = question.alternatives
+          ? question.alternatives.map((alternative) => alternative.id)
+          : [];
+        if (question.id) {
+          qids.push(question.id);
+        }
+        qids.forEach((qid) => {
+          if (qid[0] === '@') {
+            importedQids.add(qid);
+            let tids = importedQidAssessmentMap.get(qid);
+            if (!tids) {
+              tids = [];
+              importedQidAssessmentMap.set(qid, tids);
+            }
+            tids.push(tid);
+          }
+        });
+      });
+    });
+  });
+
+  if (importedQids.size > 0) {
+    let result = await sqldb.queryOneRowAsync(sql.get_course_info, { course_id: courseId });
```

Comment:
You can use `queryRow` here instead:

```suggestion
    let institutionId = await sqldb.queryRow(sql.get_course_info, { course_id: courseId }, z.string());
```

You'll need to import Zod if you accept this suggestion.

Reply:
done